src/averageAges.js

```diff
@@ -15,11 +15,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const male = century
+    ? people.filter(element => {
+      return element.sex === 'm' && Math.ceil(element.died / 100) === century;
+    })
```

Comment:
```suggestion
    ? people.filter(element => element.sex === 'm' && Math.ceil(element.died / 100) === century)
```
You may use it without 'return' here